src/Jackalope/Node.php

```diff
@@ -1462,17 +1478,6 @@ public function confirmSaved()
     }
 
     /**
-     * In addition to calling parent method, tell all properties
-     */
-    public function setPath($path, $move = false)
```

Comment:
Yeah, that's what I thought, so I've moved this logic to https://github.com/jackalope/jackalope/pull/304/files#diff-194b8b91be1497a30eb4ca5256bc194bR1623